src/careamics_restoration/bioimage/docs/n2v.md

```diff
@@ -0,0 +1,5 @@
+## Noise2Void
```

Comment:
Citation could also just be a dict returned by `get_n2v_citation`

Reply:
`n2v.md` is for the `documentation` field

Reply:
Hmm yes the build_model requires paths... In that case you are right, let's keep the md in here.